typescript/src/lib/execution/kurtosis_lambda_service_server.ts

```diff
@@ -0,0 +1,67 @@
+import { ServerUnaryCall, sendUnaryData, ServiceError, Metadata, status, handleCall, Server, ServiceDefinition } from "grpc";
+import { NetworkContext } from "kurtosis-core-api-lib";
+import { KnownKeysOnly } from "minimal-grpc-server";
+import { Result } from "neverthrow";
+import { ILambdaServiceServer } from "../../kurtosis_lambda_rpc_api_bindings/api_lambda_service_grpc_pb";
+import { ExecuteArgs, ExecuteResponse } from "../../kurtosis_lambda_rpc_api_bindings/api_lambda_service_pb";
+import { newExecuteResponse } from "../consturctor_calls";
+import { KurtosisLambda } from "../kurtosis_lambda/kurtosis_lambda";
+
+class KurtosisLambdaServiceError implements ServiceError {
+    readonly code?: status;
```

Comment:
wouldn't it be better to write getters for these, then you can make these `private readonly`

Reply:
Yep and I actually tried to do this at first, but `ServiceError` (which comes from `grpc`) doesn't have these as private so I can't :(